src/main/java/no/nav/familie/integrasjoner/dokarkiv/metadata/Dokumentmetadata.kt

```diff
@@ -36,3 +36,96 @@ enum class Dokumentkategori(private val beskrivelse: String) {
+    Dokumenttype.KONTANTSTØTTE_INNHENTE_OPPLYSNINGER_ETTER_SØKNAD_I_SED -> KontantstøtteInnhenteOpplysningerEtterSøknadISedMetadata
+    Dokumenttype.KONTANTSTØTTE_INNHENTE_OPPLYSNINGER -> KontantstøtteInnhenteOpplysningerMetadata
+    Dokumenttype.KONTANTSTØTTE_OPPHØR -> KontantstøtteOpphørMetadata
+    Dokumenttype.KONTANTSTØTTE_SVARTIDSBREV -> KontantstøtteSvartidsbrevMetadata
+    Dokumenttype.KONTANTSTØTTE_EØS -> KontantstøtteSøknadEØSMetadata
+    Dokumenttype.KONTANTSTØTTE_TILBAKEKREVING_BREV -> KontantstøtteTilbakebetalingBrevMetadata
+    Dokumenttype.KONTANTSTØTTE_TILBAKEKREVING_VEDTAK -> KontantstøtteTilbakebetalingVedtakMetadata
+    Dokumenttype.KONTANTSTØTTE_VARSEL_OM_REVURDERING_FRA_NASJONAL_TIL_EØS -> KontantstøtteVarselOmRevurderingFraNasjonalTilEøsMetadata
+    Dokumenttype.KONTANTSTØTTE_VARSEL_OM_REVURDERING -> KontantstøtteVarselOmRevurderingMetadata
+    Dokumenttype.KONTANTSTØTTE_VARSEL_OM_VEDTAK_ETTER_SØKNAD_I_SED -> KontantstøtteVarselOmVedtakEtterSøknadISedMetadata
+    Dokumenttype.KONTANTSTØTTE_VEDLEGG -> KontantstøtteVedleggMetadata
+    Dokumenttype.KONTANTSTØTTE_VEDTAK_AVSLAG -> KontantstøtteVedtakAvslagMetadata
+    Dokumenttype.KONTANTSTØTTE_VEDTAK_INNVILGELSE -> KontantstøtteVedtakInnvilgetMetadata
+    Dokumenttype.KONTANTSTØTTE_VEDTAK -> KontantstøtteVedtakMetadata
+    Dokumenttype.OVERGANGSSTØNAD_BLANKETT -> OvergangsstønadBlankettMetadata
+    Dokumenttype.OVERGANGSSTØNAD_BLANKETT_SAKSBEHANDLING -> OvergangsstønadBlankettSaksbehandlingMetadata
+    Dokumenttype.OVERGANGSSTØNAD_ETTERSENDING -> OvergangsstønadEttersendingMetadata
+    Dokumenttype.OVERGANGSSTØNAD_FRITTSTÅENDE_BREV -> OvergangsstønadFrittståendeBrevMetadata
+    Dokumenttype.OVERGANGSSTØNAD_SØKNAD -> OvergangsstønadSøknadMetadata
+    Dokumenttype.OVERGANGSSTØNAD_TILBAKEKREVING_BREV -> OvergangsstønadTilbakebetalingBrevMetadata
+    Dokumenttype.OVERGANGSSTØNAD_TILBAKEKREVING_VEDTAK -> OvergangsstønadTilbakebetalingVedtakMetadata
+    Dokumenttype.OVERGANGSSTØNAD_SØKNAD_VEDLEGG -> OvergangsstønadVedleggMetadata
+    Dokumenttype.SKJEMA_ARBEIDSSØKER -> ArbeidsregistreringsskjemaMetadata
+    Dokumenttype.SKOLEPENGER_BLANKETT_SAKSBEHANDLING -> SkolepengerBlankettSaksbehandlingMetadata
+    Dokumenttype.SKOLEPENGER_ETTERSENDING -> SkolepengerEttersendingMetadata
+    Dokumenttype.SKOLEPENGER_FRITTSTÅENDE_BREV -> SkolepengerFrittståendeBrevMetadata
+    Dokumenttype.SKOLEPENGER_SØKNAD -> SkolepengerSøknadMetadata
+    Dokumenttype.SKOLEPENGER_TILBAKEKREVING_BREV -> SkolepengerTilbakebetalingBrevMetadata
+    Dokumenttype.SKOLEPENGER_TILBAKEKREVING_VEDTAK -> SkolepengerTilbakebetalingVedtakMetadata
+    Dokumenttype.SKOLEPENGER_VEDLEGG -> SkolepengerVedleggMetadata
+    Dokumenttype.VEDTAKSBREV_BARNETILSYN -> BarnetilsynVedtaksbrevMetadata
+    Dokumenttype.VEDTAKSBREV_OVERGANGSSTØNAD -> OvergangsstønadVedtaksbrevMetadata
+    Dokumenttype.VEDTAKSBREV_SKOLEPENGER -> SkolepengerVedtaksbrevMetadata
+}
```

Comment:
kan eventuelt bytte ut dette med 
```suggestion
fun Dokumenttype.tilMetadata(): Dokumentmetadata = hentAlleDokumentMedadataKlasser().single { it.dokumenttype == this }
```

Men da får vi ikke den samme typesikringen. Det er dekket av tester, så det skal ikke ha så mye å si, men jeg synes at typesikring >> tester

Reply:
Enig! Det har jo vist seg at testene (uten typesikring), egentlig ikke har hjulpet oss noe til å huske på dette her når vi legger til noe nytt